### DISCUSSION TITLE
How to add target="_blank" at all links by default

### DISCUSSION THREAD

**Discussion by giokallis** [facebook/lexical#4639] [Q&A] [answered]
Hello people,

I want to add to my Editor the functionality that all links will open in another tab. Is it possible somehow to add by default `target={_blank}`

I searched at documentation but I didn't find anything about it.

**Comment by moy2010** [comment]
Yes, it's possible. You just have to add the attribute when the link node is created. Check how the playground has implemented the autolink plugin, for example.

**Reply by giokallis** [reply]
The truth is that I can't find much info from the playground!
What I want is for all Editor links to be saved with `target="_blank"`. 
So when they are rendered the links keep their attributes.

**Comment by giokallis** [comment, marked as answer]
Finally I found it, and it is what I want!
I leave it here for anyone else with the same worry. 
I replaced `editor.dispatchCommand(TOGGLE_LINK_COMMAND, linkUrl); ` with
`editor.dispatchCommand(TOGGLE_LINK_COMMAND, { url: linkUrl, target: '_blank' });` at the point which I set the link to editor!
